contrib/openjdk17/template.py

```diff
@@ -126,26 +123,27 @@ def do_configure(self):
     gnu_configure.configure(self, sysroot=False)
 
 
-def do_install(self):
-    _jdkp = self.cwd / "build/images/jdk"
-    if _bootstrap:
-        # first make a copy
-        bdirn = f"openjdk-bootstrap-{pkgver}-{self.profile().arch}"
-        self.mkdir(bdirn)
-        for f in _jdkp.iterdir():
-            self.cp(f, bdirn, recursive=True)
-        # remove src, we don't need it
-        self.rm(self.cwd / bdirn / "lib/src.zip")
-        # strip libs
-        for f in (self.cwd / bdirn).rglob("*.so"):
-            print("STRIP", f.relative_to(self.cwd))
-            self.do("llvm-strip", f.relative_to(self.cwd))
-        # make an archive
-        self.do("tar", "cvJf", f"{bdirn}.tar.xz", bdirn)
-        self.error("build done, collect your tarball in builddir")
+@custom_target("bootstrap", "build")
+def _bootstrap(self):
+    # first make a copy
+    bdirn = f"openjdk-bootstrap-{pkgver}-{self.profile().arch}"
+    self.mkdir(bdirn)
+    for f in (self.cwd / "build/images/jdk").iterdir():
+        self.cp(f, bdirn, recursive=True)
+    # remove src, we don't need it
+    self.rm(self.cwd / bdirn / "lib/src.zip")
+    # strip libs
+    for f in (self.cwd / bdirn).rglob("*.so"):
+        print("STRIP", f.relative_to(self.cwd))
+        self.do("llvm-strip", f.relative_to(self.cwd))
+    # make an archive
+    self.do("tar", "cvJf", f"{bdirn}.tar.xz", bdirn)
```

Comment:
make this use the zst thing from go/template.py too

Reply:
or since that is just really slow for no reason

```
self.do("tar", "cvf", f"{bdirn}.tar", bdirn)
self.do("zstd", f"-T{self.make_jobs}", "-19", "--rm", f"{bdirn}.tar")
```

Reply:
hm nvm now is the same, so yeah just copy the same thing

Reply:
works?

Reply:
yea, tested that locally too